tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/configurator/CDIConfiguratorTest.java

```diff
@@ -30,8 +30,8 @@
 @JBossServer(state=ServerRequirementState.PRESENT)
 public class CDIConfiguratorTest extends AbstractConfiguratorsTest{
 	
-	public static final String CDI_API_VERSION1_1="1.1";
-	public static final String CDI_API_VERSION1_0="1.0-SP4";
+	public static final String CDI_API_VERSION_1_2="1.2";
+	public static final String CDI_API_VERSION_2_0="2.0";
 	public static final String SEAM_FACES_VERSION="3.0.0.Alpha3";
 	public static final String SEAM_INTERNATIONAL_VERSION="3.0.0.Alpha1";
 	public static final String DELTASPIKE_CORE_API_VERSION="0.3-incubating";
```

Comment:
What happen, if we use new version of deltaspike?

Reply:
Fixed.